tests/test_client.py

```diff
@@ -536,104 +647,115 @@ def test_index_create_with_duplicate_acl_authz(swg_index_client):
         "hashes": {"md5": "8b9942cf415384b27cadf1f4d2d682e5"},
     }
 
-    r = swg_index_client.add_entry(data)
-    result = swg_index_client.get_entry(r.did)
-    assert result.acl == ["a", "b"]
-    assert result.authz == ["x", "y"]
+    res = client.post("/index/", json=data, headers=user)
+    rec = res.json
+    result = client.get("/index/" + rec["did"])
+    record = result.json
+    assert record["acl"] == ["a", "b"]
+    assert record["authz"] == ["x", "y"]
 
 
-def test_index_create_with_invalid_did(swg_index_client):
+def test_index_create_with_invalid_did(client, user):
     data = get_doc()
 
     data["did"] = "3d313755-cbb4-4b0fdfdfd8-899d-7bbac1f6e67dfdd"
-
-    with pytest.raises(ApiException) as e:
-        swg_index_client.add_entry(data)
-        assert e.status == 400
+    response = client.post("/index/", json=data, headers=user)
+    assert response.status_code == 400
 
 
-def test_index_create_with_prefix(swg_index_client):
+def test_index_create_with_prefix(client, user):
     data = get_doc()
     data["did"] = "cdis:3d313755-cbb4-4b08-899d-7bbac1f6e67d"
 
-    r = swg_index_client.add_entry(data)
-    assert r.did == "cdis:3d313755-cbb4-4b08-899d-7bbac1f6e67d"
+    response = client.post("/index/", json=data, headers=user)
+    record = response.json
+    assert record["did"] == "cdis:3d313755-cbb4-4b08-899d-7bbac1f6e67d"
 
 
-def test_index_create_with_duplicate_did(swg_index_client):
+def test_index_create_with_duplicate_did(client, user):
     data = get_doc()
     data["did"] = "3d313755-cbb4-4b08-899d-7bbac1f6e67d"
 
-    swg_index_client.add_entry(data)
-
-    with pytest.raises(ApiException) as e:
-        swg_index_client.add_entry(data)
-        assert e.status == 400
+    response = client.post("/index/", json=data, headers=user)
+    response = client.post("/index/", json=data, headers=user)
+    assert response.status_code == 400
 
 
-def test_index_create_with_file_name(swg_index_client):
+def test_index_create_with_file_name(client, user):
     data = get_doc()
     data["file_name"] = "abc"
 
-    r = swg_index_client.add_entry(data)
-    r = swg_index_client.get_entry(r.did)
-    assert r.file_name == "abc"
+    res = client.post("/index/", json=data, headers=user)
+    rec = res.json
+    res = client.get("/index/" + rec["did"])
+    rec = res.json
+    assert rec["file_name"] == "abc"
 
 
-def test_index_create_with_version(swg_index_client):
+def test_index_create_with_version(client, user):
     data = get_doc()
     data["version"] = "ver_123"
 
-    r = swg_index_client.add_entry(data)
-    r = swg_index_client.get_entry(r.did)
-    assert r.version == data["version"]
+    res = client.post("/index/", json=data, headers=user)
+    rec = res.json
+    res = client.get("/index/" + rec["did"])
+    rec = res.json
+    assert rec["version"] == data["version"]
 
 
-def test_index_create_blank_record(swg_index_client):
+def test_index_create_blank_record(client, user):
```

Comment:
```suggestion
def test_create_blank_record_with_baseid(client, user):
```
to match tests `test_create_blank_record` and `test_create_blank_record_with_file_name`